cgroups/fs/memory.go

```diff
@@ -94,24 +95,53 @@ func (s *MemoryGroup) GetStats(path string, stats *cgroups.Stats) error {
 		}
 		stats.MemoryStats.Stats[t] = v
 	}
+	stats.MemoryStats.Cache = stats.MemoryStats.Stats["cache"]
 
-	// Set memory usage and max historical usage.
-	value, err := getCgroupParamUint(path, "memory.usage_in_bytes")
+	memoryUsage, err := getMemoryData(path, "")
 	if err != nil {
-		return fmt.Errorf("failed to parse memory.usage_in_bytes - %v", err)
+		return err
 	}
-	stats.MemoryStats.Usage = value
-	stats.MemoryStats.Cache = stats.MemoryStats.Stats["cache"]
-	value, err = getCgroupParamUint(path, "memory.max_usage_in_bytes")
+	stats.MemoryStats.Usage = memoryUsage
+	swapUsage, err := getMemoryData(path, "memsw")
+	if err != nil && !strings.Contains(err.Error(), "no such file or directory") {
```

Comment:
This error check can be changed to `os.IsNotExist(err)`. Also I think its cleaner if we move that check inside `getMemoryData` so its look like:
```go
value, err := getCgroupParamUint(path, usage)
if err != nil {
  if os.IsNotExist(err) {
    return cgroups.MemoryData{}, nil
  }
  return cgroups.MemoryData{}, fmt.Errorf("...")
}
```
Then the consumer wont have to check for that error and just set the value to default value.

Wdyt ?